src/amo/components/AddonDetail.js

```diff
@@ -74,6 +89,7 @@ export class AddonDetailBase extends React.Component {
       const label = isPreviewingTheme ? i18n.gettext('Cancel preview') : i18n.gettext('Tap to preview');
       const imageClassName = 'AddonDetail-theme-header-image';
       const headerImage = <img alt={label} className={imageClassName} src={previewURL} />;
+      const buttonIsDisabled = !clientSupportsAddons();
```

Comment:
Reading `button is disabled if client does not support add-ons` reads like a double negative and was harder for my brain to grok. I guess `buttonEnabled` or `installationEnabled` here, then `disabled={!buttonEnabled}` is just as confusing and double-negative though. `disabled` is an annoying attribute 🤷‍♀️